schema.graphql

```diff
@@ -38,6 +38,7 @@ type TokenEntity @entity {
   updatedAt: DateTime!
   createdAt: DateTime!
   count: Int!
+  totalCount: Int!
```

Comment:
please use same wording as collection has

`count` for total
`supply` for available

![Screenshot 2023-10-06 at 12 04 10](https://github.com/kodadot/stick/assets/22471030/8b089010-6fdc-40f9-b4c6-b98c4fccd1ae)

Reply:
ok
so supply is only not burned
count is all

yes?

Reply:
> yes?

YES